src/appkit.cs

```diff
@@ -15137,9 +15442,11 @@ partial interface NSView : NSDraggingDestination, NSAnimatablePropertyContainer,
 		void DragImage (NSImage anImage, CGPoint viewLocation, CGSize initialOffset, NSEvent theEvent, NSPasteboard pboard, NSObject sourceObj, bool slideFlag);
 
 		[Export ("dragFile:fromRect:slideBack:event:")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use 'BeginDraggingSession (NSDraggingItem [] items, NSEvent evnt, [Protocolize] NSDraggingSource source)' instead")]
 		bool DragFile (string filename, CGRect aRect, bool slideBack, NSEvent theEvent);
 		
 		[Export ("dragPromisedFilesOfTypes:fromRect:source:slideBack:event:")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use BeginDraggingSession '(NSDraggingItem [] items, NSEvent evnt, [Protocolize] NSDraggingSource source)' with an NSFilePromiseProvider instead")]
```

Comment:
Also wrong first apostrophe, it should be before `BeginDraggingSession` not before the (

Please put `NSFilePromiseProvider` between apostrophes.